content/v2.0/reference/cli/influx/stacks/update.md

```diff
@@ -30,6 +31,16 @@ influx stacks update [flags]
 | `-u` | `--template-url`      | Template URLs to associate with a stack                    | list of strings |                       |
```

Comment:
I think teh `--export-file` is missing here 🤔

Reply:
Whoops, you're right.